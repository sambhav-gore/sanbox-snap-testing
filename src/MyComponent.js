import React from "react";

export const MyComponent = () => <div>This is a custom component</div>;

export default MyComponent;
