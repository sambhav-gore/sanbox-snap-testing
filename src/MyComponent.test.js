import React from "react";
import { render } from "react-testing-library";

import MyComponent from "./MyComponent";

describe("MyComponent", () => {
  it("should render without crashing", () => {
    render(<MyComponent />);
  });

  it("should match the snapshot", () => {
    const { container } = render(<MyComponent />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
