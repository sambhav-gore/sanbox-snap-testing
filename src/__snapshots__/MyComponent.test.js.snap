// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should match the snapshot 1`] = `
<div>
  This is a custom component
</div>
`;
